RecoLocalCalo/HcalRecAlgos/interface/HLTv2.h

```diff
@@ -0,0 +1,63 @@
+#ifndef HLTv2_h
+#define HLTv2_h 1
```

Comment:
I see lots of changes to make... I will begin applying suggested changes soon. 
Should I just push the changes to this branch again when they are done?